packages/playwright-core/src/utils/tar/extract.js

```diff
@@ -0,0 +1,311 @@
+const { Writable, Readable, getStreamError } = require('stream')
```

Comment:
`getStreamError` is not a thing. How is it supposed to work?

Reply:
Good find! Removed the usage of it by moving `_predestroy` into `_destroy`. Once I add the diff this will make more sense.